### ISSUE TITLE
Unsquashfs fails on Proxmox VE 7.0 installer

### ISSUE THREAD

**Issue by aronsky** [proxmox/pve-installer#5]
TL;DR: unsquashfs of pve-base.squashfs fails, the problem seems to be with grub-related files inside.

I tried installing Proxmox VE 7.0 (my first foray into Proxmox). After the installation failed due to exit code one from unsquashfs (when unsquashing pve-base.squashfs), I proceeded to retry several times.

I then replaced the target installation media, thinking that maybe the target drive I'm using has problems - to no avail. I thought that maybe the installation media itself was corrupted, so I tried several USB sticks, using both Rufus and balenaEtcher to create them - again, to no avail. The installation would fail at the same place every time.

I verified the sha256 of the ISO I downloaded, and it was correct.

At that point, I decided to check the squashfs file for extraction errors on the computer I used to create the installation media. I only had the ISO available to me (since the installation media partitions aren't recognized by Windows), so I looked at PVE_BOOT.SQU. It extracted fine.

I proceeded to boot Proxmox installer in debug mode, and found the pve-base.squashfs file. Interestingly, while being the same size as PVE_BASE.SQU, its sha256 was different - I'm not sure who's responsible for changing it, but that must be the core of the issue. When trying to unsquashfs pve-base.squashfs from the command line, I could see that it failed to process files related to grub.

**Comment by deanwoodward** [comment]
Same issue here, though I hadn't gotten quite as far into debugging it.

**Comment by naetimus** [comment]
Is there a solution for this? I'm having the same problem nearly a year later on a Dell R710 with a PERC H700 RAID controller. Doesn't happen on other systems.

**Comment by naetimus** [comment]
My eventual solution was to install off a DVD image, not a flash drive. Tried 3 different methods to create the flash drive, none worked. Worked first try with a DVD of the image.

**Comment by deanwoodward** [comment]
I'll have to burn a DVD and try from there... thanks for the idea. (My lab is torn down pending move to a new location so it'll be at least a month before I even have a chance to try again, but since it's been a year since I first asked... no rush.)

**Comment by aronsky** [comment]
Unfortunately, I don't remember what I did originally to resolve the issue
for myself - but it didn't involve a DVD, as I don't have a DVD drive in
the computer I used...

**Comment by WhiteNight121** [comment]
Hello I had this issue as well it looks like DVD did the trick as well going to add all the info i gathered in hopes of closing this out.

System: Dell R710
i updated the BIOS to 6.6 (newest i could find)

i reset the RAID settings just in case but im not sure if that helped out not.
i also saw that after the first proxmox install screen the monitor on the server went black and i had to use a workaround to even see the errors ect.

this is the error when it was still trying to install 
![Capture1](https://user-images.githubusercontent.com/49812570/196011417-053a675a-e8d4-44d2-8444-92d8b2e09653.JPG)

it looks like if i waited a min or 2 i would get pasted the error above and then i would get thought the proxmox wizard and when it got to 100% it would fail with this error.
![Capture2](https://user-images.githubusercontent.com/49812570/196011447-82f0174c-fba0-49fa-97db-cd0da1ce7ab7.JPG)

i also noted that now that proxmox is installed the monitor on the system still does not work, not sure if this is related or not.
this was my workaround https://github.com/DomiStyle/docker-idrac6 so i could see what was going on when the monitor on the server went black.

let me know if there is other info that would help.
